pubspec.yaml

```diff
@@ -16,8 +16,8 @@ dependencies:
 
   receive_sharing_intent:
     git:
-      url: git@github.com:krabbenprgr/receive_sharing_intent.git
-      ref: receive-.txt-v2
+      url: https://github.com/sherlockvn/receive_sharing_intent.git
+      ref: master
```

Comment:
Pls use lib in linagora repo

https://github.com/linagora/receive_sharing_intent

Reply:
done